src/main/java/git/Ui.java

```diff
@@ -73,6 +78,8 @@ public String[] processInput() {
 
     /**
      * Prompts user for expiration date.
+     *
+     * @return the formatted expiration date
```

Comment:
I think the first word after @return should be capitalised as stated in the se-edu guides! Same for other parameters like @param